This is a temporary hack to disconnect some demo programs from the
build process. This is because fluid2 wants to establish an X server
connection when building these programs and fails when it cannot
do so.

The issue is known and probably a fluid2 bug, see FLTK2 STR #1302.
--- test/Makefile.orig	Fri Jul  7 19:38:14 2006
+++ test/Makefile	Fri Jul  7 19:38:14 2006
@@ -156,7 +156,6 @@
 	pack$(EXEEXT) \
 	pixmap$(EXEEXT) \
 	progress$(EXEEXT) \
-	radio$(EXEEXT) \
 	qubix$(EXEEXT) \
 	resizable$(EXEEXT) \
 	resizealign$(EXEEXT) \
@@ -170,7 +169,6 @@
 	timer$(EXEEXT) \
 	utf$(EXEEXT) \
 	fullscreen$(EXEEXT) \
-	valuators$(EXEEXT) \
 	list$(EXEEXT) \
 	wizard$(EXEEXT)
 
